### DISCUSSION TITLE
Release 8 - April 2022

### DISCUSSION THREAD

**Discussion by sinecurelife** [PaulJerimy/SecCertRoadmapHTML#55] [Announcements]
- Added 61 certifications
- Removed 18 retired certifications
- Corrected broken links and outdated pricing
- Renamed the acronym for some certifications for easier searching
- Added proficiency levels to the left for easier orientation
- Added 2 columns to accommodate more certifications
- Added 1 row to accommodate more certifications
- Changed tooltip behavior
- Added tooltips to the domain heading text
- Adjusted blues and yellows to improve contrast between domains

<hr /><em>This discussion was created from the release <a href='https://github.com/PaulJerimy/SecCertRoadmapHTML/releases/tag/'>Release 8 - April 2022</a>.</em>

**Comment by sawft99** [comment]
I noticed for Mile2 certs you have C)ISSM lower than C)ISSO which gives the impression that it is easier and I know that the difficulty is semi vague. However, I notice that Mile2 lists C)ISSM after C)ISSO making it seem like that is the progression they recommend.

https://www.mile2.com/information_systems_security_auditor_outline/

Same for C)ISSM and C)ISMS-LA/LI

https://www.mile2.com/cisms-la-li-outline/

**Reply by fuadstrykerlab** [reply]
Agreed. Mile2 advertise C)ISSM as the certification path after C)ISSO.
